docs/topics/native/apple-privacy-manifest.md

```diff
@@ -54,25 +46,43 @@ valid values for each field.
 
-## Tool for searching usages of required reason API in dependencies
+## Finding required reason APIs in the dependencies of your Kotlin framework
+
+Kotlin code in your app may access required reason APIs from libraries such as `platform.posix`. For example if you Kotlin code or one the dependencies accesses API such as `fstat`:
+```kotlin
+import platform.posix.fstat
+
+fun useRequiredReasonAPI() {
+  fstat(...)
+}
+```
 
 In some cases, it may be hard for you to find out from which dependency the required reason API usage comes from. To assist you, we've built a simple tool.
 
-To use it, run the following command in the directory of your project where Kotlin framework declaration (TODO not sure if it's comprehensible) is located
-```python3 -c "$(curl -fsSL https://github.com/JetBrains/kotlin/raw/rrf_v0.0.1/libraries/tools/required-reason-finder/required_reason_finder.py)"```
-You also may download the script separately, inspect it, and run it via `python3`.
+To use it, run the following command in the directory of your project where the Kotlin framework is declared:
+```
+/usr/bin/python3 -c "$(curl -fsSL https://github.com/JetBrains/kotlin/raw/rrf_v0.0.1/libraries/tools/required-reason-finder/required_reason_finder.py)"
+```
+You may also download the script separately, inspect it, and run it via `python3`.
+
+## Placing PrivacyInfo.xcprivacy in your Kotlin artifacts
+
+Use the following instructions if you need to bundle `PrivacyInfo.xcprivacy` file with your Kotlin: 
+
+* If you produce an XCFramework from your Kotlin code, write the `PrivacyInfo.xcprivacy` file and manually place it in the [framework](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/adding_a_privacy_manifest_to_your_app_or_third-party_sdk#4336740) within the XCFramework bundle
+* If you integrate Kotlin code using a local integration with `embedAndSignAppleFrameworkForXcode` task, add a [local](https://developer.apple.com/documentation/xcode/organizing-your-code-with-local-packages) Swift Package with an empty target to represent your Kotlin code and add the privacy manifest to the target per [documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/adding_a_privacy_manifest_to_your_app_or_third-party_sdk#4336749)
+* If you integrate Kotlin code using a local integration with Kotlin generated CocoaPods podspec file, specify the path to `PrivacyInfo.xcprivacy` using the [resource_bundles](https://guides.cocoapods.org/syntax/podspec.html#resource_bundles) key in [extraSpecAttributes](https://kotlinlang.org/docs/native-cocoapods-dsl-reference.html#cocoapods-block) property
 
 ### Known usages
 
 #### Compose Multiplatform
 
-Compose Multiplatform rendering engine, Skia, is using `stat` and `fstat` symbols at the moment. Please follow [the issue](https://github.com/JetBrains/compose-multiplatform/issues/4738) for updates.
```

Comment:
Why was the link to Compose GH removed? Because it doesn't provide any useful info at the moment?

Reply:
I was rewriting this section and thought that specifying the APIs is enough. Thinking about it now: let's keep the Compose link.